src/laplace.jl

```diff
@@ -251,7 +252,7 @@ function optimize_prior!(
     # Setup:
     logP₀ = isnothing(λinit) ? log.(unique(diag(la.P₀))) : log.([λinit])     # prior precision (scalar)
     logσ = isnothing(σinit) ? log.([la.σ]) : log.([σinit])                   # noise (scalar)
-    opt = Adam(lr)                                                           # Adam is a stochastic gradient descent (SGD) optimization algorithm, using lr as learning rate
+    opt = Adam(lr)                                                           # Adam is gradient descent (GD) optimization algorithm, using lr as learning rate
```

Comment:
Oh, I believe minibatch gradient descent is still considered to be stochastic -- but nevermind, my comment was meant simply as a comment